fw/http_frame.c

```diff
@@ -680,20 +699,30 @@ tfw_h2_conn_streams_cleanup(TfwH2Ctx *ctx)
  * Stream linkage protection.
  */
 static inline void
-__tfw_h2_stream_add_closed(TfwClosedQueue *hclosed_streams, TfwStream *stream)
+__tfw_h2_stream_add_tfw_closed_queue(TfwClosedQueue *queue, TfwStream *stream)
 {
 	if (!list_empty(&stream->hcl_node))
 		return;
 
-	list_add_tail(&stream->hcl_node, &hclosed_streams->list);
-	++hclosed_streams->num;
+	list_add_tail(&stream->hcl_node, &queue->list);
+	stream->queue = queue;
+	++stream->queue->num;
```

Comment:
Shouldn't we move the 3 lines to a function, which is opposite to `__tfw_h2_stream_del_tfw_closed_queue()` to make the code more consistent?

Reply:
`__tfw_h2_stream_add_tfw_closed_queue` is opposite to `__tfw_h2_stream_add_tfw_closed_queue` and it have such three lines of code
```
list_del_init(&stream->hcl_node);
	--stream->queue->num;
	stream->queue = NULL;
```